Algorithm.Python/BasicTemplateFuturesFrameworkAlgorithm.py

```diff
@@ -0,0 +1,97 @@
+﻿# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Algorithm.Framework")
+AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Orders import *
+from QuantConnect.Securities import *
+from QuantConnect.Algorithm import *
+from QuantConnect.Algorithm.Framework import *
+from QuantConnect.Algorithm.Framework.Alphas import *
+from QuantConnect.Algorithm.Framework.Portfolio import *
+from QuantConnect.Algorithm.Framework.Selection import *
+from Alphas.ConstantAlphaModel import ConstantAlphaModel
+from Selection.FutureUniverseSelectionModel import FutureUniverseSelectionModel
+from Execution.ImmediateExecutionModel import ImmediateExecutionModel
+from Risk.NullRiskManagementModel import NullRiskManagementModel
+from datetime import date, timedelta
+
+### <summary>
+### Basic template futures framework algorithm uses framework components
+### to define an algorithm that trades futures.
+### </summary>
+class BasicTemplateFuturesFrameworkAlgorithm(QCAlgorithmFramework):
+
+    def Initialize(self):
+
+        self.UniverseSettings.Resolution = Resolution.Minute
+
+        self.SetStartDate(2013, 10, 7)
+        self.SetEndDate(2013, 10, 11)
+        self.SetCash(100000)
+
+        # set framework models
+        self.SetUniverseSelection(FrontMonthFutureUniverseSelectionModel(self.SelectFutureChainSymbols))
+        self.SetAlpha(ConstantFutureContractAlphaModel(InsightType.Price, InsightDirection.Up, timedelta(1)))
+        self.SetPortfolioConstruction(SingleSharePortfolioConstructionModel())
+        self.SetExecution(ImmediateExecutionModel())
+        self.SetRiskManagement(NullRiskManagementModel())
+
+
+    def OnOrderEvent(self, fill):
+        self.Log(f"{self.UtcTime}:: {fill}")
+
+    def OnSecuritiesChanged(self, changes):
+        pass#self.Log(f"{self.UtcTime}:: {changes}")
```

Comment:
I think we should either use the logging here or remove the method (same for C#)